deployment/release-s3.sh

```diff
@@ -12,15 +12,15 @@ make release
 
 ./deployment/build-example-html.sh "key_live_hcnegAumkH7Kv18M8AOHhfgiohpXq5tB" "https://api2.branch.io" "https://cdn.branch.io/branch-latest.min.js"
 aws s3 cp example.html s3://branch-builds/example.html
+aws s3 cp example.html s3://branch-cdn/example.html
 
-aws s3 cp --content-type="text/javascript" --content-encoding="gzip" dist/build.min.js.gz s3://branch-cdn/branch-$VERSION.min.js --acl public-read
-aws s3 cp --content-type="text/javascript" --content-encoding="gzip" dist/build.min.js.gz s3://branch-cdn/branch-latest.min.js --acl public-read
+aws s3 cp --content-type="text/javascript" --content-encoding="gzip" dist/build.min.js.gz s3://branch-cdn/branch-$VERSION.min.js --cache-control "max-age=300"
+aws s3 cp --content-type="text/javascript" --content-encoding="gzip" dist/build.min.js.gz s3://branch-cdn/branch-latest.min.js --cache-control "max-age=300"
```

Comment:
Is this 300 days? This policy should be documented, however unlikely

Reply:
300s

Reply:
im not sure where to document it, but i looke dat the current files in prod and they had this on the cache control